package/etc/update-motd.d/30-services

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# set column width
+COLUMNS=3
+# colors
+green="\e[1;32m"
+red="\e[1;31m"
+undim="\e[0m"
+
+services=("appdaemon@homeassistant" "home-assistant@homeassistant" "mosquitto")
+# sort services
+IFS=$'\n' services=($(sort <<<"${services[*]}"))
+unset IFS
+
+service_status=()
+# get status of all services
+for service in "${services[@]}"; do
+    service_status+=($(systemctl is-active "$service"))
+done
+
+out=""
+for i in ${!services[@]}; do
+    # color green if service is active, else red
+    if [[ "${service_status[$i]}" == "active" ]]; then
+        out+="${services[$i]}:,${green}${service_status[$i]}${undim},"
```

Comment:
```suggestion
        out+="${services[$i]}:,${green}${service_status[$i]}${undim}\\n"
```
![image](https://user-images.githubusercontent.com/15093472/47371890-e8b91300-d6e8-11e8-9836-ecfe2adc8f53.png)

If all suggestions are accepted the service part will look like this list, which IMO looks cleaner :)